app/views/users/_up_event_card.html.erb

```diff
@@ -0,0 +1,25 @@
+<% @upcoming_events.each do |event| %>
+<div class="col-md-4">
+<div class="text-center shadow-lg rounded bg-light py-2">
+<h4 class="text-info font-weight-bold">TITLE</h4>
+<span class="font-weight-normal text-dark"><%= event.title %></span>
+<h4 class="text-info font-weight-bold">description</h4>
+<p><%= event.description %></p>
+<h4 class="text-info font-weight-bold">date</h4>
+<p><%= event.time %></p>
+<div class="w-100 bg-secondary text-light py-2 mt-3 font-weight-bold"><%= link_to 'See Attendees', event %></div>
+<div class="w-100 bg-secondary text-light py-2 mt-1 font-weight-bold">Created by: <%= User.find(event.creator_id).username %></div>
```

Comment:
- [ ] Put the `User.find(event.creator_id).username` in a helper the models shouldn't be directly called in views